data/english.js

```diff
@@ -1,10 +1,44 @@
 export const texts = {
   1: {
     text:
-      'I have a pen. I have an apple. Uh! Apple-pen. I have a pen. I have pineapple. Uh! Pineapple-pen. Apple-pen... Pineapple-pen... Uh!! Pen-pineapple-apple-pen.'
```

Comment:
Can we not deleting the old texts for now. We want to have as much texts as possible. We could either hide the `Attribution` section if a text doesn't have either `url` or `name`.